### PR TITLE
Service: Wind direction

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for JACDAC.

### CODE REVIEW

File: services/winddirection.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Wind direction
+
+A sensor that measures wind direction.
+
+    identifier: 0x186be92b
+    extends: _sensor
+
+## Registers
+
+    ro wind_direction: u16 ° @ reading
```
Comment (mmoskal) [inline services/winddirection.md:10]: maybe overkill but u16.16 for future-proofing?

File: services/winddirection.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+The direction of the wind.
+
+    rw direction_offset?: i16 ° @ 0x80
+
+Offset added to direction to account for sensor calibration.
+
+    const direction_resolution?: u16 ° @ reading_resolution
```
Comment (mmoskal) [inline services/winddirection.md:18]: should it also have error?

File: services/winddirection.md (hunk 1 above)
Comment (pelikhan) [reply services/winddirection.md:10]: does not hurt... but the spark fun sensor precision was around 40deg

File: services/winddirection.md (hunk 2 above)
Comment (pelikhan) [reply services/winddirection.md:18]: yeah they come in pair

Comment (pelikhan) [reply services/winddirection.md:18]: use error,. drop resolution

File: services/winddirection.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    identifier: 0x186be92b
+    extends: _sensor
+
+## Registers
+
+    ro wind_direction: u16 ° @ reading
+
+The direction of the wind.
+
+    ro wind_direction_offset?: i16 ° @ 0x80
```
Comment (mmoskal) [inline services/winddirection.md:14]: what is this one? is it somehow detected by the sensor?

**Comment by pelikhan** [comment]
The idea of the offset would be to calibrate the direction based on the orientation of the sensor

________________________________

File: services/winddirection.md (hunk 2 above)
Comment (mmoskal) [reply services/winddirection.md]: > The idea of the offset would be to calibrate the direction based on the orientation of the sensor

but it's read-only, I'm not sure how would that work

**Pull request merged**